examples/cpp/core/scoped_allocator/allocator.h

```diff
@@ -12,12 +12,6 @@ namespace scoped_allocator
     {
         using value_type = T;
 
-#if defined(_MSC_VER) && _MSC_VER < 1910
-        // MSVC 2015 and older versions have bug in std::scoped_allocator_adaptor
```

Comment:
We're still targeting MSVC 2015, so I think this need to stay.

Reply:
Yeah, good catch. Thanks!